### PR TITLE
Add one-click button to Scalingo

### PR DESCRIPTION
**Pull request by EtienneM**
I had to make it a bit more configurable through the use of two environment variables: `HOST` and `PORT`. The defaults are still the same value, respectively `localhost` and `8080`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Formbuilder
 If you want to try it out, have a look [at the demo
 page](https://kinto.github.io/formbuilder/)
 
+Or deploy it on Scalingo in a single click on this button:
```
Comment (almet) [inline README.md]: Do I need to pay in order to use this? If so we should make it visible.

**Comment by EtienneM** [comment]
OK I added it but FYI we already have a note about it in the Kinto installation documentation (http://kinto.readthedocs.io/en/stable/tutorials/install.html#deploying-on-cloud-providers).

**Comment by almet** [comment]
Thanks! But, now that I think through this a bit more: what you're offering here is to host the formbuilder as a server, whereas it can be built as a static webapp. As a result, I wonder if that "deploy to scalingo" really makes sense ? Rather than doing this, we should provide a way to configure the kinto server used from the static webapp.

What do you think?

**Comment by Natim** [comment]
Thanks

**Comment by Natim** [comment]
According to the code, this will host the formbuilder webapp and ask you to configure the SERVER_URL when building it.

**Pull request merged**